src/auth/googleauth.ts

```diff
@@ -131,6 +132,14 @@ const GoogleAuthExceptionMessages = {
     'Unable to detect a Project Id in the current environment. \n' +
     'To learn more about authentication and Google APIs, visit: \n' +
     'https://cloud.google.com/docs/authentication/getting-started',
+  NO_CREDENTIALS_FOUND:
+    'Unable to find credentials in current environment. \n' +
+    'To learn more about authentication and Google APIs, visit: \n' +
+    'https://cloud.google.com/docs/authentication/getting-started',
+  NO_UNIVERSE_DOMAIN_FOUND:
+    'Unable to detect a Universe Domain in the current environment.\n' +
```

Comment:
nit: I think this message could be a little clearer - the link it points to doesn't mention universe domains anywhere, not sure if that is something that will be updated in the near future. Otherwise, maybe just make it clear that the universe domains is meant to be retrieved via the metadata keys assuming that's what the user would need to set up.

Reply:
Once the `universe` predefined key has been made available it will be linked there